tools/arm_pack_manager/index.json

```diff
@@ -462674,5 +462674,45 @@
         ], 
         "vendor": "Nordic Semiconductor:54"
     }, 
+    "LPC55S69JBD100": {
+    "core": "Cortex-M33",
+    "vendor": "Cypress",
+    "sectors": [
+        [
+            262144,
+            512
+        ]
+    ],
+    "memories": {
+        "IRAM1": {
+            "access": {
+                "execute": false,
+                "non_secure": false,
+                "non_secure_callable": false,
+                "peripheral": false,
+                "read": true,
+                "secure": false,
+                "write": true
+            },
+            "default": true,
+            "size": "0x22000",
+            "start":"0x20022000",
+            "startup": false
+        },
+        "IROM1": {
+            "access": {
+                "execute": true,
+                "non_secure": true,
+                "non_secure_callable": false,
+                "peripheral": false,
+                "read": true,
+                "secure": false,
+                "write": true
+            },
+            "start": "0x00040000",
+            "size": "0x58000"
+        }
+      }
+    },
```

Comment:
We have PR 10068 for this, can you please remove this section

Reply:
@mmahadevan108 AFAIK #10068 is not destined for 5.12, while this one is, so I can't have this PR depend on it.

Reply:
@david, is the `index.json` here same as in #10068?
@mmahadevan108 as this PR more urgent for rc3, is it possible to make #10068 dependent on this PR?

Reply:
Not the same. I can make it the same (which is the largest part of that PR).

Reply:
@davidsaada @mmahadevan108 Then I suggest you sync with each other

Reply:
#10068 is labelled release-version-5.12.0-rc3. Should be merged soon.

Reply:
Oh OK then. So will make this one depend on it.

Reply:
Please delete this change from this PR.

Reply:
@mmahadevan108 This certainly needs to be deleted. However, as I explained below, our tools won't build the bootloader well without this change. This should currently be on hold until resolved.

Reply:
👍 . Would you like to try with the changes from #10068 to ensure it works fine with the tools.

Reply:
Sorry if I wasn't clear - this requires a change in the bootloader build tools, by which index.json should be ignored in such cases. Until then, this PR will fail if we use the values from #10068. In other words, this PR should be on hold until these tools changes are introduced.

Reply:
👍 , got it.